hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestAssignmentManagerOnCluster.java

```diff
@@ -598,6 +605,68 @@ public void testCloseFailed() throws Exception {
     }
   }
 
+  /**
+   * This tests region close with exponential backoff
+   */
+  @Test(timeout = 60000)
+  public void testCloseRegionWithExponentialBackOff() throws Exception {
+    String table = testTableName.getTableName().toString();
```

Comment:
you already get a "TableName", you don't need to convert it to string and back to TableName. Also, TEST_UTIL.deleteTable() takes TableName as argument.